doc/.ChangeLog_template

```diff
@@ -48,45 +52,53 @@ Changes made to namelist defaults (e.g., changed parameter values):
 
 Changes to the datasets (e.g., parameter, surface or initial files):
 
-Substantial timing or memory changes: [For timing changes, can check PFS test(s) in the test suite]
+Substantial timing or memory changes:
+[For timing changes, you should at least check the PFS test in the test
+ suite, whether or not you expect timing changes: ensure that its run
+ time has not increased significantly relative to the baseline.]
 
-Notes of particular relevance for developers: (including Code reviews and testing)
+
+Notes of particular relevance for developers:
 ---------------------------------------------
 NOTE: Be sure to review the steps in README.CHECKLIST.master_tags as well as the coding style in the Developers Guide
+[Remove any lines that don't apply. Remove entire section if nothing applies.]
 
 Caveats for developers (e.g., code that is duplicated that requires double maintenance):
 
 Changes to tests or testing:
 
-CTSM testing:
 
+Testing summary:
+----------------
 [... Remove before making master tag.  Available test levels:
 
-    a) regular (must be run before handing off a tag to SEs and must be run
-     before committing a tag)
-    b) build_namelist (if namelists and/or build_system changed))
-    c) tools (only if tools are modified and no CTSM source is modified)
-    d) short (for use during development and in rare cases where only a small
-          change with known behavior is added ... eg. a minor bug fix)
+    a) regular (regular tests on normal machines if CTSM source is modified)
+    c) tools (if tools are modified)
```

Comment:
- [ ] This should be changed to b, c, d rather than c, d, e

- [ ] Does the description under tools really mean, "if *only* tools are modified"? This is where I personally find it clearer to have rules for the individual test types rather than trying to define a test level: I would assume that a test level of tools means that you only ran the tools tests and not the regular tests, but I'm not sure if that's what you're thinking. (And does a test level of "tools" imply anything about whether you ran build-namelist tests, python tests, ptclm tests? This is where the whole notion of testing levels falls down for me: We have 2**5 = 32 possible combinations of test types that were / were not run with each tag, not counting the rare possibility of "minimal", so I have trouble describing that with just a couple of levels. Maybe I'm just being pedantic, but I wanted to try to explain why test levels don't make sense to my way of thinking. Anyway, it's not worth spending more time on this. If these categories make sense to you, I'm fine with this, but to be honest, I think I'll continue to ignore this because it doesn't make a lot of sense to me.)

Reply:
OK, so I worked on it more. Fixing the numbering. And to try to explain, this is a summary of type of testing that you do, not a complete description of everything. The reason for it is to help pick what testing should be done, and also give a short summary of what was done. You should still fill out the details of what tests you actually ran. For example, I often run the build-namelist tests even if I mark it as "tools" test level. There the critical testing that needed to be run was the tools tests. But, if I ran other tests that's not bad. But, if I say it's a "regular tools" test level (so needs to run regular system tests and the tools tests) I should have run both the regular tests and the tools tests. And I should see it documented that way. I view the information on  if you ran the build-namelist tests, python tests, or ptclm tests, to be details of the testing that aren't really specified in the test level. That's why they are details that you list in turn. One of the reasons for giving the details about which tests were run -- is that I then expect those tests to work for that tag. But, I can get the general idea of what was done by looking at the test level. If I see the test level is "doc" I won't expect that you did much if any testing. If I see it was "minimal" I know I have to look at the details to see what you actually did. Since, "regular" is the usual case, I'll just expect that you at least ran the standard system tests. Having a test level for the non regular case is when this might be the most useful.

Reply:
@billsacks I'm OK if you don't use the test level and I do. Your comment on the number of factorials of testing is why I find the test level requirement summary helpful. It helps me to lay out what testing I should require for a given tag. So it's useful beforehand. And it's also a useful general summary afterwards as well so I can see if the testing done was just pretty standard or something special. I still want to also describe in detail what actual tests were run, as the level only gives the general requirements, not the specific details. Anyway, I worked on it more, and think it's described better now. And I actually added two other levels "fates" and "release" which are two levels I think will be useful to have. So hopefully this is better for everyone now.